data/structure.json

```diff
@@ -1,24 +1,29 @@
 {
   "navigation": {
     "logo": "Renome",
-    "navigationIcons": {
+    "icons": {
       "cart": {
         "alt": "Cart icon",
-        "src": "cart_icon.svg"
+        "src": "cart_icon.svg",
+        "cartStatus": "5"
```

Comment:
what is cart status?

Reply:
There is no requirements regarding logic behind it but it is supposed to display items in cart. Currently there is no possibility to add anything to cart so it stays here as a placeholder